Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3-stretch
+MAINTAINER Lars von Wedel <vonwedel@me.com>
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
Why bother?

Reply:
This is the directory for all app files within the docker container. It could be anything, also /app or whatsoever, as this directory is not visible outside of the docker container. I used the same that home-assistant uses in its docker.

Reply:
Yes okay but as you said it yourself, this is "not visible outside of the docker container". That is why I wonder why you'd even bother to change the WORKDIR.

Reply:
Seems to be good practice from looking at other Dockerfiles. As the COPY command uses . as a target, I assume it makes sense to have a "well-defined" workdir rather than just a default.